canvas/gradient_test.go

```diff
@@ -7,35 +7,30 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestNewLinearGradient(t *testing.T) {
-	// Horizontal
-	horizontal := NewLinearGradient(color.Black, color.Transparent, GradientDirectionHorizontal)
-	assert.Equal(t, horizontal.CenterOffsetX, 0.0)
-	assert.Equal(t, horizontal.CenterOffsetY, 0.0)
+func TestNewHorizontalGradient(t *testing.T) {
+	horizontal := NewHorizontalGradient(color.Black, color.Transparent)
 
-	img := horizontal.Generator(50, 5)
+	img := horizontal.Compute(50, 5)
```

Comment:
As above is this change required? The new name sounds more callable than the previous Generator... (and is a new term compared to the Raster.Generator).